src/app/edit-movie-detail/edit-movie-detail.component.ts

```diff
@@ -1,8 +1,6 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
-import { Router, ActivatedRoute } from '@angular/router';
+import { Component, OnInit, Input } from '@angular/core';
 import { MovieService } from '../movie.service';
-import { FormGroup, FormBuilder, FormControl, Validators } from "@angular/forms";
-
+import { FormGroup, FormBuilder, Validators } from "@angular/forms";
```

Comment:
Why was 'FormControl' removed?

Reply:
Looking at the files, I don't think it was being used elsewhere. Ideally we only import things we use in the file itself.